map/map_loader/README.md

```diff
@@ -6,15 +6,35 @@ This package provides the features of loading various maps.
 
 ### Feature
 
-pointcloud_map_loader loads PointCloud file and publishes the map data as sensor_msgs/PointCloud2 message.
+`pointcloud_map_loader` provides pointcloud maps to the other Autoware nodes in various configurations.
+Currently, it supports the following two types:
 
-### How to run
+- Publish raw pointcloud map
+- Send partial pointcloud map loading via ROS 2 service
 
-`ros2 run map_loader pointcloud_map_loader --ros-args -p "pcd_paths_or_directory:=[path/to/pointcloud1.pcd, path/to/pointcloud2.pcd, ...]"`
+#### Publish raw pointcloud map (ROS 2 topic)
 
-### Published Topics
+The node publishes the raw pointcloud map loaded from the `.pcd` file(s).
+
+#### Send partial pointcloud map (ROS 2 service)
+
+Here, we assume that the pointcloud maps are divided into grids.
+
+Given a query from a client node, the node sends a set of pointcloud maps that overlaps with the queried area.
+Please see [the description of `GetPartialPointCloudMap.srv`](https://github.com/autowarefoundation/autoware_msgs/tree/main/autoware_map_msgs#getpartialpointcloudmapsrv) for details.
+
+### Parameters
+
+| Name                | Type   | Description                                                                       | Default value |
+| :------------------ | :----- | :-------------------------------------------------------------------------------- | :------------ |
+| enable_whole_load   | bool   | A flag to enable raw pointcloud map publishing                                    | true          |
+| enable_partial_load | bool   | A flag to enable partial pointcloud map server                                    | true          |
+| leaf_size           | double | Downsampling leaf size (only used when enable_downsampled_whole_load is set true) | 3.0           |
```

Comment:
What is `enable_downsampled_whole_load`?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1938/commits/3bff6ecc2dd2ec2f46cc7a2d3029f5d5e401edc9 Resolved